rpc/src/v1/tests/eth.rs

```diff
@@ -16,166 +16,16 @@
 
 //! rpc integration tests.
 use std::env;
-use std::sync::Arc;
```

Comment:
Hey @jleni , can you explain why you are pulling code out of `eth.rs` and into `eth_tester.rs`?  Is it a stylistic refactor or somehow needed for the voting RPC implementation?

Reply:
No, it is not just stylistic. 
`eth.rs` contained many test helpers that were used only in the respective unit tests.
- Loading chains from json files
- Setting up mocks to test RPC requests.
- etc.
The functionality is also necessary and could be reused in the Clique unit tests.

In the long term, we should really get all these integration tests and helpers into a single place. At the moment, they seem a bit scattered in the code base.

Reply:
Nevertheless, expect changes as I work through this PR.